scripts/clean-db-dump.sql

```diff
@@ -0,0 +1,102 @@
+\echo 'Update users'
+UPDATE users
+SET name = 'Test user',
+    email_address = id || '@user.marketplace.team',
+    password = :bcrypt_password;
+
+-- Remove data about currently ongoing framework applications
+\echo 'Delete draft services for open frameworks'
+DELETE
+FROM draft_services
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+\echo 'Delete supplier frameworks for open frameworks'
+DELETE
+FROM supplier_frameworks
+WHERE framework_id IN
+    (SELECT id
+     FROM frameworks
+     WHERE status='open');
+
+-- Remove data related to open DOS procurements
```

Comment:
I don't know to be honest, I'm not super familiar with it. The SQL here is pretty much a copy/paste job from our old cleanup script.

Would it be beneficial?